js/app.js

```diff
@@ -0,0 +1,60 @@
+
+/*fetchCountries = async () => {
+  const response = await fetch('https://restcountries.com/v2/all');
+  const myJson = await response.json(); //extract JSON from the http response
+  console.log(myJson);
+  localStorage['jsonData'] = JSON.stringify(myJson);*/
+
+
+
+
+
+const flags = document.getElementById('flags')
+
+document.addEventListener("DOMContentLoaded", e => {
+  fetchData()
+})
+
+const fetchData = async () => {
+  try {
+    const res = await fetch('https://restcountries.com/v2/all')
+    //const res = await fetch('https://restcountries.com/v3.1/all') no jala con la versiòn 3.1, no sè por qué? será la sintaxis?
+    const data = await res.json()
+    ensign(data)
+    formRequest(data)
+    filters(data)
+  } catch (error) {
+    console.log(error)
+  }
+}
+
+const ensign = data => {
```

Comment:
Add indentation to this function, indentation is to add tabs on the left, so it looks cleaner and easier to read.